Mollier/SAM.Core.Mollier.UI/Controls/CustomizePointControl.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace SAM.Core.Mollier.UI.Controls
+{
+    public partial class CustomizePointControl : UserControl
+    {
+        private Color pointColor;
```

Comment:
It would be better if CustomizePointControl would store UIMollierPoint instead its properties